firebase-firestore/src/androidTest/java/com/google/firebase/firestore/POJOTest.java

```diff
@@ -216,6 +238,19 @@ public void testWriteAndRead() {
     assertEquals(data, otherData);
   }
 
+  @Test
+  public void testDocumentIdAnnotation() {
```

Comment:
Should we test a nested documentId?  E.g. you could add a`POJOWithDocumentIdAnnotation nested` field to `POJOWithDocumentIdAnnotation` (sort of a recursive data structure).

Reply:
Done.